mcp/src/main/java/io/modelcontextprotocol/server/McpAsyncServer.java

```diff
@@ -680,12 +694,23 @@ public Mono<Void> loggingNotification(LoggingMessageNotification loggingMessageN
 			return this.mcpTransportProvider.notifyClients(McpSchema.METHOD_NOTIFICATION_MESSAGE, params);
 		}
 
-		private McpServerSession.RequestHandler<Void> setLoggerRequestHandler() {
+		private McpServerSession.RequestHandler<Object> setLoggerRequestHandler() {
```

Comment:
Shouldn't it be an actual type?

Reply:
this request returns empty response. Same like Ping